packages/components/src/components/post-language-switch/post-language-switch.tsx

```diff
@@ -111,11 +110,10 @@ export class PostLanguageSwitch {
 
   private renderList() {
     return (
-      <Host
-        data-version={version}
-        aria-label={`${this.caption}, ${this.description}`}
-        aria-role="list"
-      >
+      <Host data-version={version} aria-role="list">
+        <span aria-label={`${this.caption}, ${this.description}`}>
+          {this.activeLang.toUpperCase()}
+        </span>
```

Comment:
This behavior does not match with the behavior on post.ch, the caption should at least be on the host.